app/views/layouts/_header.html.haml

```diff
@@ -0,0 +1,28 @@
+%header#site-header
+  .lg-container.container
+    = link_to root_path, class: "exercism-link" do
+      = icon "exercism-with-logo-black", "Exercism"
+
+    %nav
+      %ul
+        %li
+          = link_to "#" do
+            = graphical_icon(:dashboard)
+            Dashboard
+
+        %li.selected
```

Comment:
You'll want to add: `aria-current="page"`.

Reply:
Done. Thanks :)